modules/ossm-deploying-automatic-gateway-injection.adoc

```diff
@@ -73,9 +82,68 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: default
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy <6>
+metadata:
+  name: vs2igw
+  namespace: vs2
+spec:
+  podSelector:
+    matchLabels:
+      istio: ingressgateway
+  ingress:
+    - {}
+  policyTypes:
+  - Ingress
+---
+apiVersion: autoscaling/v2 <7>
+kind: HorizontalPodAutoscaler
+metadata:
+  labels:
+    app: vs2igw
+    istio: ingressgateway
```

Comment:
Change this label to be (to align with what us defined in the deployment):
`istio: <ingressgateway>` <2>

Reply:
done